src/components/dialog/dialog.mdx

```diff
@@ -127,6 +127,18 @@ This may occasionally be useful with things like Toasts where they persist on th
 
 <Canvas of={DialogStories.Responsive} />
 
+
+### Focusing Dialog Container Programmatically
+
+```javascript
+import { DialogHandle } from "carbon-react/lib/components/dialog";
+```
+
+The `DialogHandle` type provides an imperative handle for programmatic control over the `Dialog`'s container focus. 
```

Comment:
suggestion(non-blocking): the wording feels a bit technical and could be simplified

Feel free to push back, but I prefer to aim for more layman's terms where possible

```suggestion
The `DialogHandle` type allows you to control the focus of the Dialog component using a ref. You can call the `focus()` method on the `ref` to set the focus on the Dialog's container."
```